### DISCUSSION TITLE
Pls add the way to configure the service in the readme ..

### DISCUSSION THREAD

**Discussion by Schrebergarten03-links** [christgau/wsdd#190] [General]
Hi i´ll make it short ...

Searched for an hour how to configure the service and or wsdd ...
Just installed it ... cli comands gave errors with started service ... service didnt keep the setting form cli .. 
Im 2 years into linux as a hobby ... so not a total noob ...

So i found out that in an ubuntu setting the config file for the service is in nano /etc/default/wsdd
and well the rest is intuitive but would have been easier if that would have be written anywhere ...
Best case would be readme file XD

my problem is i a have a custom "workgroup" and it is defined in samba but wsdd is just using "workgroup" wanted to change just that ...

So thats that ... 
greetings and thx

**Comment by christgau** [comment]
> Searched for an hour how to configure the service and or wsdd ...

What I usally do, if I do not find something in the man page (not even the upstream README) is: `find /etc -name '*program_of_interest*'`. 

>  Just installed it ... cli comands gave errors with started service ... 

That's not surprising. Similar to other network services, you can run wsdd only once on a given machine. If you try otherwise, you end up with error messages.

> service didnt keep the setting form cli ..

How should it? Wsdd does not act as configuration frontend to itself. Have you seen similar applications that persist settings made by the invocation of a service binary when the service is already running? I don't think that's a popular approach in a Linux/Unix environment. 

> So i found out that in an ubuntu setting the config file for the service is in nano /etc/default/wsdd

Well, that's distro/init.d-specific. 

> and well the rest is intuitive but would have been easier if that would have be written anywhere ...
> Best case would be readme file XD

This repo only supplies example configuration files, which is documented in the README. Those files follow common practices. They also may be used by downstream packages of distros, but distros can do otherwise as well. So even if the common practices are (redundantly) documented here, the documentation might not be accurate.

In the end, I'm not going to document how service configuration does work on a certain distro/downstream package here.

**Comment by JedMeister** [comment]
> So i found out that in an ubuntu setting the config file for the service is in nano /etc/default/wsdd
and well the rest is intuitive but would have been easier if that would have be written anywhere ...
Best case would be readme file XD

Perhaps lodge a bug with Ubuntu and/or Debian? As @christgau notes, that's a distro specific location, so your issue seems to be with the package in Ubuntu (which likely comes from a Debian package, so arguably best fixed there and Ubuntu will inherit it).
